beacon_node/network/src/service.rs

```diff
@@ -631,19 +627,6 @@ impl<T: BeaconChainTypes> NetworkService<T> {
                         warn!(self.log, "Failed to update ENR"; "error" => e);
                     }
                 }
-                // if the discovery service is not auto-updating, update it with the
```

Comment:
removed on 61d4ae13af06366fb4123af081234812d2b5f3ab, is it not required?

Reply:
It was annoying to send back the UDP sockets with the added QUIC support. I realised the only reason we send back the UDP sockets was to update the ENR in this code section. After looking at it, i decided to remove it for two reasons. 

1. It was kinda annoying to send back all the udp-sockets and code was simpler this way. 
2. I don't think this behaviour is intuitive and actually what we want. The `auto-update` feature of discovery is on by default. Users should only ever remove that if that know what they are doing and they know their own external ports and that they are not going to change. So if a user has selected that option and set specific external ports, I dont think we should change those settings because UPnP may come around and give us new ports. i.e if you set auto-update to be off, your ports are static. Removing this code makes them static, which I think is how it should be.

Reply:
thanks for explaining Age, makes sense!